multinet/uploaders/csv.py

```diff
@@ -142,7 +142,12 @@ def upload(
     # Perform validation.
     validate_csv(rows, key, overwrite)
 
-    if key != "_key" and overwrite:
+    # Once we reach here, we know that the specified key field must be present,
+    # and either:
+    #   key == _key     # noqa: E800
+    #   or key != _key and the _key field is not present
+    #   or key != _key and _key is present, but overwrite = True
```

Comment:
Could you surround the instances of the word `_key` in these comments by quotation marks? Unless you mean a variable by the name of `_key` (which I don't believe is the case).

Reply:
Good point